docs/FeatureManagement/Samples/TargetingFilter.modified.sample.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
I think we should remove the `.modified` files and use a flag shaped like this to confirm Brittney and Aiden within the `TargetingFilter.sample.json`:

```json
{
    "id": "RolloutPercentageUpdate",
    "description": "A feature flag using a targeting filter, that will return true 61% of the time. Changing to 62% makes the user Brittney true.",
    "enabled": true,
    "requirement_type": "All",
    "conditions": {
        "client_filters": [
            {
                "name": "Microsoft.Targeting",
                "parameters": {
                    "Audience": {
                        "DefaultRolloutPercentage": 61
                    }
                }
            },
            {
                "name": "Microsoft.Targeting",
                "parameters": {
                    "Audience": {
                        "DefaultRolloutPercentage": 62
                    }
                }
            }
        ]
    }
}
```

Reply:
What would requirement type all validate here? It's the same as doing a single one at 62% true.